package.json

```diff
@@ -44,7 +43,6 @@
     "jest": "^25.3.0",
     "kind-of": ">=6.0.3",
     "node-mocks-http": "^1.7.6",
-    "node-sass": "^4.14.1",
```

Comment:
When Mouhajer looked he couldn't find where it was used, also as it turns out we can't use v6.0.1 of node-sass because the version of node we use as a company is out of date so we couldn't update it even if we wanted to.